octopus-firmware/README.md

```diff
@@ -1,19 +1,21 @@
 # Octopus device integration
 
 This example provides a prototype of a real world device integration - based on the Octopus board (ESP8266).
-It shows how to subscribe to Bosch IoT Suite for Asset Communication and use it's API to provision a device.
+It shows how to subscribe to Bosch IoT Suite for Asset Communication and how to use its API to provision a device.
 
 The octopus will be able to connect automatically to a given wireless internet access, to send telemetry data to the
 Things service, and to react to messages which can be sent by a [web application](../octopus-frontend).
 
-Before you can start, you have to prepare your setup. This includes the following steps:
-1. [Subscribe for the Bosch IoT Suite for Asset Communication](#setting-up-bosch-iot-suite-for-asset-communication)
-2. [Register the device via Device Provisioning API](#device-provisioning-api)
-3. [Set-up the Arduino IDE](#prepare-the-octopus-device-with-arduino)
+![](./images/octo-firmware.png)
 
-We will use the _telemetry_, _event_ as well as the _command & control_ pattern of the Bosch IoT Hub.
+The device will send telemetry data to the Things service, and will react to messages, which can be sent by a web application, e.g. [octopus-frontend](../octopus-frontend).
+
+The example is structured in following steps:
```

Comment:
As a German I perfectly understand what you mean by this, but I don't know if it's correct English. I think it is probably more like `The example is divided into the following steps`. Wdyt? (Also applies for the octopus-simulator/README.md)